lms/static/sass/elements-v2/_icons.scss

```diff
@@ -0,0 +1,31 @@
+.xseries-icon {
+  background: url('#{$static-path}/images/programs/xseries-icon.svg') no-repeat;
+}
+
+.micromasters-icon {
+  margin-top: $baseline * 0.05;
+  background: url('#{$static-path}/images/programs/micromasters-icon.svg') no-repeat;
+}
+
+.certificate-icon .certificate-body {
+  @include float(left);
+  @include margin-right($baseline*0.4);
+
+  margin-top: ($baseline/10);
+  width: 23px;
+  height: 20px;
+  padding: 2px;
+  background-color: $white;
+  border-style: solid;
+  border-width: 2px;
```

Comment:
@AlasdairSwan I think doing that makes sense once we have a need for SVG icons elsewhere. At the moment, these are only used on program detail pages, so making that change now seems premature. I'll look into making the Sass more concise.